docs/example_workflow_explained.rst

```diff
@@ -0,0 +1,316 @@
+``bmorph`` Example Workflow
+===========================
+
+This page describes how to setup data for and bias correct it through ``bmorph``. 
+If you want to save time in copying over the code discussed in this file
+you can copy the Jupyterlab Notebook ``run_bmorph_template.ipynb``.
+
+Import Packages and Load Data
+-----------------------------
+We will be using numpy, xarray, and pandas in this example notebook.
+Note numpy can imported directly intsead of using magic if desired.
```

Comment:
Users might not know what magic is - just add a reference to [this](https://ipython.readthedocs.io/en/stable/interactive/magics.html) and note that you refer to the `%pylab inline` command.